packages/react/src/theming/defaultTheming.ts

```diff
@@ -1,6 +1,6 @@
-import { theme, tokens } from '@aws-amplify/ui';
+import { theme as defaultTheme } from '@aws-amplify/ui';
 import { getCSSVariablesFromTokens } from './utils';
 
-const defaultCSSVariables = getCSSVariablesFromTokens(tokens);
```

Comment:
Changing to `.value` doesn't solve that just yet... That would be part 2. But now thinking about it, without part 2 this breaks the theme Provider even more so I might need to add those changes... But yes, right now actually our default theme typescript object contains no references because they got resolved by style dictionary at build-time. So in the example above if you overrode `colors.background.tertiary` it would not affect the alert component because at that time that token is already resolved to a specific color. You could still use `({ theme }) => theme.colors.background.tertiary` and that would work as expected, but it just wouldn't affect the style of primitives.